frodo2-runtime-android/build.gradle

```diff
@@ -1,36 +1,13 @@
-import com.android.build.gradle.LibraryPlugin
-import org.aspectj.bridge.IMessage
-import org.aspectj.bridge.MessageHandler
-import org.aspectj.tools.ajc.Main
-
-buildscript {
-  repositories {
-    jcenter()
-    maven {
-      url  "http://dl.bintray.com/android10/maven"
-    }
-    mavenLocal()
-  }
-  dependencies {
-    classpath "com.android.tools.build:gradle:$project.androidBuildToolsVersion"
-    classpath "org.aspectj:aspectjtools:$project.aspectjVersion"
-  }
-}
-
-repositories {
-  maven {
-    url  "http://dl.bintray.com/android10/maven"
-  }
-  mavenLocal()
-}
+//import com.android.build.gradle.LibraryPlugin
 
 apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
```

Comment:
what is this doing?

Reply:
This is fixing some issues with the default maven plugin that is not designed for Android